src/css/styles.scss

```diff
@@ -1,46 +1,90 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try uncommenting this and saving. What changes in the browser?
-@import "~normalize.css";
+.top {
+	display: block;
+	height: 80px;
+	margin-left: auto;
+	margin-right: auto;
+	background-color: #f8f9eb;
+	font-family: raleway light;
```

Comment:
Be sure to include a fallback font here! Also - is this font loaded with Google Fonts? I don't see it in your repo, so double check. You can try opening it on your phone to be sure - if your font isn't loaded properly, it definitely won't show there. :)